lms/static/scripts/frontend_apps/components/test/mock-imported-components.js

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
Wondering if `test/util.js` and this file could be combined? They all sorta feel like important test utils.

Reply:
I wondered about this myself. I put this code in a separate file so the imports would be consistent between tests in the lms and client apps. Lyza asked about having a clearer separation between tests and utilities used by tests. A possible structure might be something like:

```
components/test/util/
  wait.js
  mock-imported-components.js
```

Perhaps this is mature enough to move into a separate package which the client and lms apps could import (as `@hypothesis/test-utils` or something).

Thoughts?